### DISCUSSION TITLE
將手語即時翻譯成文字的新技術！方便聾啞人群溝通

### DISCUSSION THREAD

**Discussion by mesakiyuta** [1101-GS4719-C/Python-Community-GS4719-C#169] [科技新知]
> 內文

日本軟銀(Soft bank)跟電氣通信大學自2017年開始發展此技術，透過人工智慧將手語轉換成文字，好讓不會手語的人也能跟聾啞人溝通。
操作介面如圖，會用類似LINE的方式呈現，在比完手語的5、6秒之後，就會顯示螢幕上，而此軟體也同時搭載了將口語轉換成文字的功能，所以聾啞人也能夠看見對話內容。
![image](https://user-images.githubusercontent.com/90549404/143185293-62b43e01-f93e-41c9-a782-a8d584122004.png)

> 優點

不用戴上任何偵測動作的儀器，或是準備特別的背景，就能夠即時翻譯手語，設備上的需求低，只需要兩台筆電或手機，就能夠利用這項服務，在未來普及時的所需的成本低。

> 問題點

記者在實際測試時，偶而會遇到無法翻譯的問題，原因在於每個人對同一個單字的動作並不完全相同，另外手語也有地區上的不同，使動作不盡相同。
為了解決這些問題，目前最大的困境在於資料不足，一個單字最少也需要100人以上的動作資料，但要收集手語的動作資料並不容易。

> 解決案

軟銀在七月開發了一個app，專門用於收集手語動作情報，希望透過大眾的支持，使這項技術更加成熟。

> 心得

最近看了一部叫做「國王排行」的作品，劇中的主角是聾啞人，在和其他人交流時需要透過手語，但常常對方看不懂手語，無法理解他在說什麼，像這樣不能溝通帶給了主角很多的傷害。
若此項技術能夠順利地發展，像男主一樣的人，就多了一種溝通的方式，想必能減少對聾啞人的不便。

另外我看了這文章才知道原來日文手語有分兩種，一種叫「日本手話」，語法跟日文不一樣，常用於聽障者之間，而另一種是「日本語対応手話」，文法跟日文一樣，用於聽障者跟非聽障者之間，不知道中文的的手語是不是同樣多於一種? 知道的同學幫我在留言區回答一下！

**Comment by enoch-lightning** [comment]
這個技術目前只能應用於日文嗎？有中文、英文或其它語言的版本嗎？

**Reply by enoch-lightning** [reply]
🗾

**Reply by mesakiyuta** [reply]
目前沒有其他語言的版本，但我認為只要資料夠齊全應該是能應用到其他語言上

**Comment by mesakiyuta** [comment]
發現忘了附上連結，在這裡補上
https://news.yahoo.co.jp/articles/713499740f7524a2d3ba73c7d1d7c4b4b5aadb16

https://www.techbang.com/posts/91580-japan-will-use-ai-technology-to-enable-direct-text-conversion?from=home_news

**Comment by Isongzhe** [comment]
國王排行好看，推爆

**Comment by HSU-YU-JEN** [comment]
這個手語翻譯的app很棒，只是就像上面說的，如果要真正完全正確判斷手語的話，蒐集的樣本數必須要很多，才能讓AI判斷變得更準

**Comment by Jie2222** [comment]
是很棒的想法，但要建立準確的模型最重要的就是數據的收集，以目前來說可能關注的人遠遠不夠，如果有夠多的人關心支持，未來人工智慧才能幫助到更多、不同語言、國籍的聾啞人士。

**Comment by 109802503gloria** [comment]
好有趣，第一次知道原來手語也有類似地方方言的差異

**Comment by ChinChun0220** [comment]
若要多國語言，需要每個國家都要有數據，不知道平常新聞發布會的手語老師的影片能不能用呢？

**Reply by chia-kai-chang** [reply]
新聞發布時的手語老師是很棒的訓練資料來源。

**Comment by 107602505** [comment]
好奇透過人工智慧將手語轉換成文字，是類似圖像辨識的概念嗎?
若日本成功將這個技術推廣我相信應該也會有很多國家跟上~

**Comment by Jen-Siang** [comment]
好酷的科技，希望趕快發展起來用到世界各地
